examples/ch32v307/src/bin/i2c-bmp180-async.rs

```diff
@@ -0,0 +1,71 @@
+//! I2C test with SSD1306
+//!
+//! Folowing pins are used:
+//! SDA     PB11
+//! SCL     PB10
+//!
+//! Depending on your target and the board you are using you have to change the pins.
+//!
+//! For this example you need to hook up an SSD1306 I2C display.
```

Comment:
I am not sure that this line is still relevant :)

Reply:
Thanks. I'll fix this later.